apps/financial-aid/open-api/src/app/models/application.model.ts

```diff
@@ -109,4 +110,10 @@ export class ApplicationModel {
 
   @Field({ nullable: true })
   readonly navSuccess?: boolean
+
+  @Field({ nullable: true })
+  readonly childrenComment?: string
```

Comment:
How does this work. Does the applicant have many children but one comments which is affects all the children?